### PR TITLE
Quest Improvements - 26 March

### PR DESCRIPTION
**Pull request by chochem**
- Fixes 'Even more Challenge needed'. It should not be repeatable!
- Rewrite Safety First in a few parts, fix a bunch of typos and add it to the Storing and Transforming Tab (I saw that one typo fix PR and saw there were a lot more)
- add urls to safety first and your first night.

### CODE REVIEW

**Review by D-Cysteine — APPROVED** [review]
Oops, looks like there's a bug with printing multiple of the same quest in `en_US.lang`. I will fix this.

File: config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/SafetyFirst-AAAAAAAAAAAAAAAAAAAIcQ==.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
...
...h multiple amps, such as battery buffers, transformers, and diodes. Check out the quest tab Storing and Transforming EU for more on those.\n\nIf you want to send more amps, use a larger cable or a superconductor (MV+, but ULV and LV have their own lossless cables). You can use laser hatches much later to send even more. There even are wireless options, but not right now.\n\nA single block machine starved of EU will restart progress but not lose the recipe items. A multiblock machine will lose the items.\n\nNote that any block that emits EU has loss when doing so: 2 ^ (tier-1) per amp. So 1 EU at LV, 2 EU at MV, and so on. Using large numbers of battery buffers in a row is thus very wasteful of your power. You have to be careful with losses when making your first EBF later in LV, maybe rather power them directly? Of course, if you want to keep your amps topped up over longer distances, you\u0027ll need to place BBs occasionally to \u0027top up\u0027 the amps, though this is less the case the further ingame you get, as the loss becomes less of something to worry about.\n\nIf this info isn\u0027t enough for you, head on over to https://gtnh.miraheze.org/wiki/Electricity to read more."
```
[the remark this answers is not in the document]
Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/SafetyFirst-AAAAAAAAAAAAAAAAAAAIcQ==.json]: fixed that and added the URL parts, however it does nothing for me

[the remark this answers is not in the document]
Comment (chochem) [reply config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/SafetyFirst-AAAAAAAAAAAAAAAAAAAIcQ==.json]: ok I just made a separate ticket for that https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13050

[the remark this answers is not in the document]
Comment (D-Cysteine) [reply config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/SafetyFirst-AAAAAAAAAAAAAAAAAAAIcQ==.json]: Thanks! Sorry for the trouble

**Pull request merged**